<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetGroupScreen</class>
 <widget class="QWidget" name="SetGroupScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>323</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SetGroupScreen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>171</width>
       <height>231</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>1080</height>
      </size>
     </property>
     <property name="title">
      <string>Set Group</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Element Name To Set Group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelElementName">
        <property name="text">
         <string>Element Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select Group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxSelectGroup">
        <property name="maxVisibleItems">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditEditGroupName"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonEditGroupName">
        <property name="text">
         <string>Edit Group Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSetGroup">
        <property name="text">
         <string>Set Group For This Element</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Add Group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditAddNewGroup"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddNewGroup">
        <property name="text">
         <string>Add New Group</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
